storybook/src/components/accordion/accordion.css-template.ts

```diff
@@ -26,7 +26,9 @@ export const cssAccordion: ComponentImplementation<Accordion<TemplateResult>> =
           ${section.icon && accordion.reverseAlign
             ? html`<span class="dso-icon">${iconTemplate({ icon: section.icon })}</span>`
             : nothing}
-          ${section.handleTitle}
+          ${typeof section.handleTitle !== "string"
+            ? html`${renvooiTemplate({ value: section.handleTitle })}`
+            : html`${section.handleTitle}`}
```

Comment:
Er schiet mij nog iets te binnen, volgens mij mag dit ook:

```suggestion
          ${typeof section.handleTitle !== "string"
            ? renvooiTemplate({ value: section.handleTitle })
            : section.handleTitle
```

Wat vinden jullie, duidelijker?

Reply:
Aangepast